utils/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var User = require.main.require('./src/user');
+var utils = module.exports;
+
+// gets a User object from username, returns only data with only 1 user object
+// which matches the username param exactly
+utils.usernameSearch = function (username, callback) {
+	User.search(
```

Comment:
This is a horrible way to search by username. Much better to use a direct DB call.

Reply:
Changed